libraries/botbuilder-dialogs-adaptive/src/actions/replaceDialog.ts

```diff
@@ -9,14 +9,24 @@ import { DialogTurnResult, DialogContext, TurnPath } from 'botbuilder-dialogs';
 import { BaseInvokeDialog } from './baseInvokeDialog';
 import { BoolExpression } from 'adaptive-expressions';
 
+/**
+ * Action which calls another dialog, when it is done it will go to the callers parent dialog.
```

Comment:
I think it is caller's, probably I'm wrong.
```suggestion
 * Action which calls another dialog, when it is done it will go to the caller's parent dialog.
```